yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -176,6 +192,9 @@ func buildCommandArguments(dbConfig AssessMigrationDBConfig, exportDirPath strin
 	return args
 }
 
+/*
+Sample header: <dbtype>,<hostname>,<port>,<service_name>,<sid>,<tns_alias>,<username>,<password>,<schema>
```

Comment:
where is dbname? I might have missed this in the previous PR. Did you chose to use "service_name" instead of this? That is an oracle-specific detail, so I think we shouldn't change our interface altogether. 
We should have db_name for now, not service_name. (consistent with all the other comands). When we revisit these args, we can change them across the board. Till then we should be consistent.

Reply:
done